src/frontend/csrs-portal/src/app/components/welcome-user/welcome-user.component.ts

```diff
@@ -1,79 +1,50 @@
 import { Component, OnInit } from '@angular/core';
-import { map } from 'rxjs/operators';
-import { JsonPipe } from '@angular/common';
 import { AccountService } from 'app/api/api/account.service';
-import { LookupService } from 'app/api/api/lookup.service';
 import { LoggerService } from '@core/services/logger.service';
-import { of } from 'rxjs';
-import { Inject, Injectable, Optional }                      from '@angular/core';
-import { HttpClient, HttpHeaders, HttpParams, HttpStatusCode,
-         HttpResponse, HttpEvent, HttpParameterCodec }       from '@angular/common/http';
-import { Observable }                                        from 'rxjs';
+import { Inject} from '@angular/core';
+import { HttpClient, HttpStatusCode, HttpResponse } from '@angular/common/http';
 import { AccountFileSummary } from 'app/api/model/accountFileSummary.model';
-import { Router, ActivatedRoute } from '@angular/router';
+import { Router } from '@angular/router';
 import { AppRoutes } from 'app/app.routes';
-
-
 @Component({
   selector: 'app-welcome-user',
   templateUrl: './welcome-user.component.html',
   styleUrls: ['./welcome-user.component.scss']
 })
 export class WelcomeUserComponent implements OnInit {
-  _logger: LoggerService;
   _reponse: HttpResponse<AccountFileSummary>;
 
   constructor(@Inject(AccountService) private accountService,
               @Inject(LoggerService) private logger,
               @Inject(Router) private router) {}
 
   ngOnInit(): void {
-    this._logger = this.logger;
-
 
     this.accountService.apiAccountGet('response', false).subscribe({
       next: (data) => {
         this._reponse = data;
         if (this._reponse.status === HttpStatusCode.NotFound) {
-          //redirect to empty application form
+          //redirect to application form
           this.router.navigate(AppRoutes.routePath(AppRoutes.APPLICATIONFORM));
         }
-        if (this._reponse.status === HttpStatusCode.Ok) {
-          if (this._reponse.body.files === null || this._reponse.body.files.length === 0)
-          {
-            //redirect to empty application from
-            this.router.navigate(AppRoutes.routePath(AppRoutes.APPLICATIONFORM));
-          }
-          else
-          {
-            if (this._reponse.body.files.length > 0)
+        else
```

Comment:
Hey @skiselev1 
Good day.
As discussed during the call please combine the two sections. Thank you so much Sergey!